application/server/execute_command.go

```diff
@@ -31,7 +31,7 @@ import (
 
 func executeCommandHandler(srv *jrpc2.Server) jrpc2.Handler {
 	return handler.New(func(ctx context.Context, params sglsp.ExecuteCommandParams) (any, error) {
-		// The context provided by the JSON-RPC server is cancelled once a new message is being processed,
+		// The context provided by the JSON-RPC server is canceled once a new message is being processed,
```

Comment:
Same here, let's not go down this road: https://www.merriam-webster.com/grammar/canceled-or-cancelled

Reply:
This ambiguity is a great example of the value this lint rule brings; both are spellings are correct. I'm happy to let the rule decide (even if it's not how I normally write "cancelled" :) ).